.github/workflows/build.yml

```
@@ -22,10 +22,17 @@ jobs:
     matrix:
       php: ["8.0", "7.4"]
       symfony: ["^4.4", "^5.2"]
-      sylius: ["~1.10.0"]
+      sylius: ["~1.10.0", "~1.11.0"]
       node: ["^14.17.x"]
       mysql: ["8.0"]
 
+      exclude:
+          - sylius: ~1.10.0
+            symfony: 4.4
+
+          - sylius: ~1.11.0
+            php: 7.4
+
```

```suggestion
      exclude:
          - sylius: ~1.11.0
            php: 7.4
```

There's no reason to exclude this version matrix.